.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+# See reference docs at
+# https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions
+name: ci
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Clone the repo
+        uses: actions/checkout@v2
+      # https://github.com/golangci/golangci-lint-action
+      - name: Lint
+        uses: golangci/golangci-lint-action@v1
+        with:
+          version: v1.26
```

Comment:
travis uses v1.21.0.

maybe update travis too, and fix the lints by disabling the newly added default inters in `.golangci.yml` (some of the new linters are really stupid lol!), or downgrade this for now?

Reply:
done. updated to v1.27.0, same as in the bb app repo.